src/TuDa.CIMS.Api/Controllers/AssetItemController.cs

```diff
@@ -22,9 +23,9 @@ public AssetItemController(IAssetItemService assetItemService)
     /// </summary>
     /// <returns> a 200 OK response if the operation is successfully and a 400 BadRequest response if any error occurs </returns>
     [HttpGet]
-    public async Task<IActionResult> GetAllAsync()
+    public async Task<IActionResult> GetAllAsync([Query] string? nameOrCas)
```

Comment:
🔧 Unfortunately, that's the wrong attribute. This is from `Refit,` but we need the `FromQuery` from Asp.Net
```suggestion
    public async Task<IActionResult> GetAllAsync([FromQuery] string? nameOrCas)
```